### PR TITLE
add sign-up page

### PR DESCRIPTION
**Pull request by hsnice16**
Implemented the Sign Up page for e-commerce.

Met the following requirement - 

image

Please review this PR, and give your feedback.

Live link - https://inspiring-brahmagupta-c9920d.netlify.app/signup.html

This is how the page looks

image

### CODE REVIEW

**Review by vrx29 — APPROVED** [review]
Looks good to me.

**Review by BalajiHarinath — APPROVED** [review]
Looks Good to me.

**Review by samyukthasudhakar — APPROVED** [review]
The code is well documented and it looks perfect. Great work Himanshu !

File: app.css [hunk 1]
```diff
@@ -63,3 +63,17 @@ header {
...
 }
+
+/** Utilities **/
+
+.w-4 {
+  width: 4rem;
+}
+
+.w-38 {
+  width: 45rem;
```
Comment (venkateshwarans) [inline app.css:74]: Keep the width and the class name proportional.

**Review by venkateshwarans — COMMENTED** [review]
All good. Nice and clean. :)

**Pull request merged**